nebula-exchange/src/main/scala/com/vesoft/nebula/exchange/processor/Processor.scala

```diff
@@ -113,26 +113,40 @@ trait Processor extends Serializable {
           dateTimeValue = rowValue.trim.split(" ")
         } else {
           throw new UnsupportedOperationException(
-            s"wrong format for datetime value: $rowValue, correct format is 2020-01-01T12:00:00:0000")
+            s"wrong format for datetime value: $rowValue, " +
+              s"correct format is 2020-01-01T12:00:00:0000 or 2020-01-01 12:00:00:0000")
```

Comment:
need to tell the user, the datatime is the UTC time, no local time. it user wants to use local_time. need config timezone.

Reply:
add note in readme